src/style.css

```
@@ -0,0 +1,37 @@
+body {
+  margin: initial;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  color: #e5e5e5;
+}
+
+.stars--1 > .stars__star:nth-child(-n+1) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-child(-n+2) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--3 > .stars__star:nth-child(-n+3) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--4 > .stars__star:nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--5 > .stars__star:nth-child(n) {
+  background-image: url(/images/star-active.svg);
+}
```

combine into one rule
```suggestion

.stars--3 > .stars__star:nth-child(-n+3),
.stars--4 > .stars__star:nth-child(-n+4),
.stars--5 > .stars__star:nth-child(n) {
  background-image: url(/images/star-active.svg);
}
```